File: src/ActiveFertilizer.java
public class ActiveFertilizer {
    public final Fertilizer fertilizer;
    public int remainingDuration;

    public ActiveFertilizer(Fertilizer fertilizer, int remainingDuration) {
        this.fertilizer = fertilizer;
        this.remainingDuration = remainingDuration;
    }

    public ActiveFertilizer copy() {
        return new ActiveFertilizer(fertilizer, remainingDuration);
    }

    @Override
    public String toString() {
        return fertilizer.name + " (" + remainingDuration + ")";
    }
}

File: src/ActiveFertilizer.py
# ActiveFertilizer.java

class ActiveFertilizer:
    def __init__(self, fertilizer, remaining_duration):
        self.fertilizer = fertilizer
        self.remaining_duration = remaining_duration

    def copy(self):
        return ActiveFertilizer(self.fertilizer, self.remaining_duration)

    def __str__(self):
        return f"{self.fertilizer.name} ({self.remaining_duration})"

